pvlib/iotools/srml.py

```diff
@@ -222,3 +228,89 @@ def read_srml_month_from_solardat(station, year, month, filetype='PO',
     url = "http://solardat.uoregon.edu/download/Archive/"
     data = read_srml(url + file_name, map_variables=map_variables)
     return data
+
+
+def get_srml(station, start, end, filetype='PO', map_variables=True,
+             url="http://solardat.uoregon.edu/download/Archive/"):
```

Comment:
Thinking more about minor consistency details, I guess we could standardize the order of `map_variables` and `url`.  Looking through other functions, here's what I see:
- `map_variables` first: `get_cams`
- `url` first: `get_psm3`, `get_pvgis_hourly`, `get_pvgis_tmy`

#1767 currently does `map_variables` first but of course can be changed easily.  

I don't see one as much better than the other.  I guess I'd favor `map_variables` first since I think it is very uncommon that people will want to mess with `url`, but maybe `map_variables=False` has some somewhat common uses.  @AdamRJensen what do you think?

Reply:
I actually thought about discussing this for your PR, but I guess I don't have a strong opinion except for consistency. But the logic of having URL last makes sense to me - it's just more of a hassle changing this as there are more existing functions with URL first.

I opened #1791 based on this discussion fyi.